aoc2021hs.cabal

```diff
@@ -83,19 +83,3 @@ executable aoc2021hs
 
     -- Base language which the package is written in.
     default-language: Haskell2010
-
```

Comment:
pls dont yell at me i promise ill write tests at some point, that's just a lot to take on rn while still trying to learn base language features